File: api/dashboard/Dashboard/Models/SessionDashboardData.cs
namespace Dashboard.Models;

public class SessionDashboardData
{
    public int Id { get; set; }
    public int PaymentAmount { get; set; }
    public int NumberOfPayments { get; set; }
    public int WrongAnswers { get; set; }
    public int CorrectAnswers { get; set; }
    public int Questions { get; set; }
}

File: ui/dashboard/src/app/entities/session-dashboard-data.ts
import { TimestampedAnswer } from './answer-dto';

export interface SessionDashboardData {
  id: number;
  dashboardId: string;
  paymentAmount: number;
  numberOfPayments: number;
  wrongAnswers: number;
  correctAnswers: number;
  questions: number;
  answers?: TimestampedAnswer[];
}
